configs/multifactor_huddle_config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Interdisciplinary Huddle",
+  "leaderID": "1",
+  "days": [1],
+  "lookAhead": 14,
```

Comment:
Looks ahead 14 huddles (i.e., 14 weeks in this case, which is one more week than the max time between huddles for the lowest score).